### ISSUE TITLE
induced by eating cheese

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6168]
**Preferred term label:**
Rearrange the triggered by xxx to group the food ingestion terms. Consider the best way to include a term for this disease.

https://omim.org/entry/602079

https://hpo.jax.org/app/browse/term/HP:0025204
**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
